docs/guide/signing-data.md

```diff
@@ -133,6 +133,7 @@ signTypedDataV4Button.addEventListener('click', function (event) {
       // Defining the chain aka Rinkeby testnet or Ethereum Main Net
       chainId: 1,
       // Give a user friendly name to the specific contract you are signing for.
+      // This is primairly for UX purposes and is needed for the signature.
```

Comment:
spell check: `primairly` -> `primarily`

Reply:
oh, looks like this line was maybe from https://github.com/MetaMask/metamask-docs/pull/277?